src/components/pages/StudentHome/StudentSidebar.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Menu, Layout } from 'antd';
+import 'antd/dist/antd.css';
+import {
+  DashboardOutlined,
+  CheckCircleOutlined,
+  MessageOutlined,
+  GroupOutlined,
+  StarOutlined,
+  SlidersOutlined,
+  ExportOutlined,
+  ReadOutlined,
+} from '@ant-design/icons';
+import { useOktaAuth } from '@okta/okta-react';
+
+const { Sider } = Layout;
+
+const StudentSideBar = props => {
+  const [collapsed, setCollapsed] = useState(false);
+  const { authService } = useOktaAuth();
+  const onCollapse = () => {
+    if (collapsed === true) {
+      setCollapsed(false);
+    } else {
+      setCollapsed(true);
+    }
+  };
+
+  return (
+    <Sider
+      data-testid="sider"
+      theme="light"
+      collapsible
+      collapsed={collapsed}
+      onCollapse={onCollapse}
+    >
+      <Menu
+        className="parent-dashboard-sidebar"
```

Comment:
I'm seeing a lot of references to the `ParentSidebar` here - in fact, these links match 1-to-1 with that sidebar and don't make sense within their current context here! Let's swap these routes out even if we haven't built those components yet so that it's clearer what work needs to be done from here and makes sense to read.

Reply:
This has been resolved in the new PR: https://github.com/BloomTech-Labs/coder-heroes-fe/pull/216